cli/tests/test_auth.rs

```diff
@@ -155,7 +180,9 @@ fn test_auth_login_existing_default() {
 
     let temp_dir = tempfile::tempdir().unwrap().into_path();
     write_first_creds(&temp_dir);
+    assert_mode(&temp_dir.join("credentials.toml"), 0o100644);
```

Comment:
644 is correct here since we're using the `write_first_creds` helper to initially create the file. I think keeping that as-is is helpful, as it lets us confirm that we update the permissions existing files.

Reply:
I think it's worth a comment in that case

Reply:
Since we're no longer updating the permissions of existing files, I updated the helper to create the file as 600.